openfisca_core/taxbenefitsystems.py

```diff
@@ -202,6 +236,9 @@ def neutralize_column(self, column_name):
         self.update_column(column_name, neutralize_column(self.get_column(column_name)))
 
     def add_legislation_params(self, path_to_xml_file, path_in_legislation_tree = None):
+        """
+        Adds an xml file to the legislation parameters.
```

Comment:
xml => XML

Tell path_in_legislation_tree is a list of names describing the destination node of the legislation tree